src/firebase.js

```diff
@@ -0,0 +1,20 @@
+// Import the functions you need from the SDKs you need
+import { initializeApp } from "firebase/app";
+// TODO: Add SDKs for Firebase products that you want to use
+// https://firebase.google.com/docs/web/setup#available-libraries
+
+// Your web app's Firebase configuration
+const firebaseConfig = {
+  apiKey: "AIzaSyAFp8PWGDN0heJCg0VJxuVpsrvbIeASaXw",
+  authDomain: "project-3---react-app.firebaseapp.com",
+  databaseURL: "https://project-3---react-app-default-rtdb.firebaseio.com",
+  projectId: "project-3---react-app",
+  storageBucket: "project-3---react-app.appspot.com",
+  messagingSenderId: "706940626968",
+  appId: "1:706940626968:web:a421e5b16904791dae7b6d"
+};
+
+// Initialize Firebase
+const app = initializeApp(firebaseConfig);
```

Comment:
When exporting the firebase object, consider naming it "firebase" rather than "app" to avoid confusion when you import it into the App.js file. If you decide to do so, make sure to update the respective import/export statements.

Reply:
Funny you touched on this - when Adrian had gone through this with us, I was confused why he imported 'firebase' and not 'app' when he had exported 'app.' I felt they should stay consistent, but I stuck to the lessons for this one. I noticed you had called it 'firebase' to avoid confusion which gave me the 'boost' to change it.